scripts/translate_bundles.py

```
@@ -0,0 +1,121 @@
+        self._setup_translator()
+
+    def _setup_translator(self):
+        """Installs the translation model if not present."""
+        argostranslate.package.update_package_index()
+        available = argostranslate.package.get_available_packages()
+        pkg = next((p for p in available if p.from_code == self.src and p.to_code == self.tgt), None)
+        if pkg:
+            print(f"--- Installing model {self.src} -> {self.tgt} ---")
+            argostranslate.package.install_from_path(pkg.download())
+        else:
+            raise ValueError(f"Translation package not available for {self.src} -> {self.tgt}. "f"Available languages: {sorted(set(p.to_code for p in available if p.from_code == self.src))}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)